tests/llmobs/test_experimentation_config.py

```diff
@@ -0,0 +1,306 @@
+import os
+from unittest.mock import patch, call
+
+import pytest
+
+import ddtrace.llmobs.experimentation._config as config
+from ddtrace.llmobs.experimentation.utils._exceptions import ConfigurationError
+
+
+# --- Fixture to Reset Global Config State ---
+
+@pytest.fixture(autouse=True)
+def reset_config_state():
+    """Resets global configuration state before each test."""
+    # Reset global state variables to defaults
+    config._IS_INITIALIZED = False
```

Comment:
these values aren't kept in sync with the defaults, definitely want to move away from the globals